server/cpho/models/metadata.py

```diff
@@ -31,11 +31,69 @@ class IndicatorDataSubmission(models.Model):
 
 
 @add_to_admin
-class PhacOrgRole(models.Model):
+class IndicatorDirectory(models.Model):
+    name = models.CharField(max_length=100)
+    description = models.TextField(blank=True)
+
+    users = fields.ManyToManyField(
+        "cpho.User",
+        through="cpho.IndicatorDirectoryUserAccess",
+        related_name="indicator_directories",
+    )
+
+    indicators = fields.ManyToManyField(
+        "cpho.Indicator",
+        through="cpho.IndicatorDirectoryLink",
+        related_name="indicator_directories",
+    )
+
+    def __str__(self):
+        return self.name
+
+
+@add_to_admin
+class IndicatorDirectoryLink(models.Model):
+    """
+    through model between an indicator and an indicator directory.
+    """
+
+    indicator = fields.ForeignKey(
+        "cpho.Indicator",
+        on_delete=models.CASCADE,
+        related_name="indicator_directory_links",
+    )
+    directory = fields.ForeignKey(
+        "cpho.IndicatorDirectory",
+        on_delete=models.CASCADE,
+        related_name="indicator_directory_links",
+    )
+
+    class Meta:
+        unique_together = ("indicator", "directory")
+
+    def __str__(self):
+        return f"{self.directory} - {self.indicator}"
+
+
+@add_to_admin
+class IndicatorDirectoryUserAccess(models.Model):
+    """
+    through model between an indicator directory and a user
+    """
+
     user = fields.ForeignKey(
-        "cpho.User", on_delete=models.CASCADE, related_name="phac_org_roles"
+        "cpho.User",
+        on_delete=models.CASCADE,
+        related_name="indicator_directory_user_accesses",
     )
-    phac_org = fields.ForeignKey(
-        "cpho.PHACOrg", on_delete=models.CASCADE, related_name="phac_org_roles"
+    directory = fields.ForeignKey(
+        "cpho.IndicatorDirectory",
+        on_delete=models.CASCADE,
```

Comment:
I'm trying something a bit weird here: I'm defining explicit through models for the m2m relations. This allows us to be flexible and add m2m fields on the user or indicator models, making it extra-easy to define forms. 

Note that the directory isn't a versioned model, so I'm not sure if versioning works properly here. If we ever try explicit through-models on a versioned model, we'll want explicit tests to check everything is versioning properly.